app/routes/_gcn.docs.circulars.markdown.mdx

```diff
@@ -0,0 +1,307 @@
+---
+handle:
+  breadcrumb: Markdown
+---
+
+import { feature } from '~/lib/env.server'
+import { SyntaxReference } from '~/routes/_gcn.circulars.edit.$circularId/CircularEditForm.tsx'
+
+export async function loader() {
+  if (!feature('CIRCULARS_MARKDOWN')) throw new Response(null, { status: 404 })
+  return null
+}
+
+# Astro-Flavored Markdown
+
+## Markdown Syntax
+
+Markdown formatting can be accessed in the quick menu of the markdown bar, but it can also be added manually.
```

Comment:
This is not sufficiently descriptive of where to find this in the user interface.